TrafficCapture/nettyWireLogging/src/main/java/org/opensearch/migrations/trafficcapture/netty/LoggingHttpRequestHandler.java

```diff
@@ -126,6 +131,12 @@ public void channelInactive(ChannelHandlerContext ctx) throws Exception {
     }
 
     protected void channelFinishedReadingAnHttpMessage(ChannelHandlerContext ctx, Object msg, HttpRequest httpRequest) throws Exception {
+        MDC.put("channelId",ctx.channel().id().asLongText());
+        MDC.put("httpMethod", httpRequest.method().toString());
+        MDC.put("httpEndpoint", httpRequest.uri());
+        metricsLogger.info("Full request received");
+        MDC.remove("httpMethod");
+        MDC.remove("httpEndpoint");
```

Comment:
Did you intentionally not remove the channelId?  There's no reason to here either (& when there is, it probably won't work right or be testable - so it's likely never going to be valuable for us)